### DISCUSSION TITLE
unknown error: Code: UNAVAILABLE server does not allow insecure connections, client must use SSL/TLS

### DISCUSSION THREAD

**Discussion by ninetynin** [planetscale/discussion#454] [Bug Reports] [answered]
```
✦ ❯ diesel setup
Creating migrations directory at: /home/anand/F/random-practice-learning/dbms-practice/te1/bird_watcher_api/migrations
Creating database: bird_db
unknown error: Code: UNAVAILABLE
server does not allow insecure connections, client must use SSL/TLS
```

This is the error im getting when im trying to run diesel setup in terminal for setting up rust database does anyone has any idea how to fix this? And yeah nothing is showing up here too
![image](https://user-images.githubusercontent.com/91336664/235287862-078c8134-0749-4e00-a8f7-c4ce12debffa.png)

Im following this tutorial .. https://planetscale.com/blog/build-a-rust-api-with-rocket-diesel-mysql

**Comment by tbarn** [comment, marked as answer]
Hi @ninetynin! We've happy to help you figure it out. What is your operating system? 

It is having issues with the certificate that allows us to do SSL/TLS connections. This can depend on the operating system. https://planetscale.com/docs/concepts/secure-connections

**Reply by ninetynin** [reply]
Hey, My os is arch(manjaro i3wm) I'll try again in future if i face this issue again Im not using planetscale and switched as i need to have foreign keys thanks 👍

**Reply by tbarn** [reply]
Okay, in the future, make sure that the `ca-certificates` package is installed. For Arch, it is likely store at: `/etc/ssl/certs/ca-certificates.crt`. This is in the docs at that link if you need it in the future.

**Reply by ninetynin** [reply]
yep thanks
